src/templates/workshop-single.js

```diff
@@ -0,0 +1,57 @@
+// import React from "react"
```

Comment:
I think this file is not needed. Can it be removed ?
@kishore007k , @Tanuj22 @Jashpatel1 ?